### ISSUE TITLE
NextImage loader doesn't correctly set the width and height params

### ISSUE THREAD

**Issue by mg-aceik** [Sitecore/jss#1646]
### Describe the Bug

NextImage loader doesn't correctly set the width and height params on the url depending on the fill value. This results in Sitecore image processor to add black borders to the image depending on its size.

It returns srcset with values like `/-/jssmedia/test.jpg?h=1705&amp;iar=0&amp;w=2740&amp;rev=7a67deeaa84446e69d7bbafbe2d4baf4&amp;mw=640`

### To Reproduce

    <NextImage
      field={props.fields.image}
      fill 
      sizes="100vw" />

### Expected Behavior

srcset with values like `/-/jssmedia/test.jpg?iar=0&amp;rev=7a67deeaa84446e69d7bbafbe2d4baf4&amp;mw=640`
In other words, strip out the height and width params.

### Possible Fix

Update the [sitecoreLoader ](https://github.com/Sitecore/jss/blob/d24e8ac7815289f0f5b9e69c853f88224f16bc43/packages/sitecore-jss-nextjs/src/components/NextImage.tsx#L19) to remove these params if the fill param is passed, similar to how it is done on the [props themselves](https://github.com/Sitecore/jss/blob/d24e8ac7815289f0f5b9e69c853f88224f16bc43/packages/sitecore-jss-nextjs/src/components/NextImage.tsx#L90). 

### Provide environment information

* Sitecore Version: XM Cloud
* JSS Version: 21.5.0

**Comment by Juansereina** [comment]
I'm experiencing the same issue. I've written a custom loader to remove the `h` and `w` properties, and I use it in the image component when I add the fill attribute.

```ts
// src/lib/sitecore-loader-override

import { ImageLoader, ImageLoaderProps } from 'next/image';

export const sitecoreLoader: ImageLoader = ({
  src,
  width,
}: ImageLoaderProps): string => {
  const [root, paramString] = src.split('?');
  const params = new URLSearchParams(paramString);
  params.set('mw', width.toString());

  // custom change
  params.delete('h');
  params.delete('w');

  return `${root}?${params}`;
};
```

```tsx
import { sitecoreLoader } from 'lib/sitecore-loader-override';

<NextImage
  field={props.fields.image}
  fill
  loader={sitecoreLoader}
/>
```

**Comment by illiakovalenko** [comment]
@mg-aceik Thank you for submitting your contribution! I've added a ticket containing all the details. It has been added to our internal backlog for thorough review
